manual/assets/js/src/demos/blur.ts

```diff
@@ -142,15 +151,19 @@ window.addEventListener("load", () => void load().then((assets) => {
 	};
 
 	const p0 = tab.pages[0];
-	tab.pages[0].addBinding(gaussianBlurPass.fullscreenMaterial, "kernelSize", { options: gaussKernels });
+	p0.addBinding(gaussianBlurPass.fullscreenMaterial, "kernelSize", { options: gaussKernels });
 	p0.addBinding(gaussianBlurPass.fullscreenMaterial, "scale", { min: 0, max: 2, step: 0.01 });
-	p0.addBinding(gaussianBlurPass.resolution, "scale", { label: "resolution", min: 0.5, max: 1, step: 0.05 });
+	p0.addBinding(gaussianBlurPass.resolution, "scale", { label: "resolution", min: 0.5, max: 1, step: 0.01 });
 	p0.addBinding(gaussianBlurPass, "iterations", { min: 1, max: 8, step: 1 });
 
-	// const p1 = tab.pages[1];
-	// p1.addBinding(kawaseBlurPass.fullscreenMaterial, "kernelSize", { options: KernelSize });
-	// p1.addBinding(kawaseBlurPass.fullscreenMaterial, "scale", { min: 0, max: 2, step: 0.01 });
-	// p1.addBinding(kawaseBlurPass.resolution, "scale", { label: "resolution", min: 0.5, max: 1, step: 0.05 });
+	const p1 = tab.pages[1];
+	// TODO: is it ok that the following two scale parameters are independant from each other?
```

Comment:
Not sure about this: "is it ok that the following two scale parameters are independent from each other?"